doc/contributing/documentation_guide.md

```diff
@@ -328,6 +328,18 @@ Alternatives:
     - Example {source}[https://github.com/ruby/ruby/blob/34d802f32f00df1ac0220b62f72605827c16bad8/doc/contributing/glossary.md?plain=1].
     - Corresponding {output}[https://docs.ruby-lang.org/en/master/contributing/glossary_md.html].
 
+### Constants
+
+In discussions, and especially in example code,
+prefer using a constant as a value (rather than a numeric, string, or symbol)
+whenever there is a choice.
```

Comment:
I can't really think of much occurences of that beyond encodings. But perhaps you do?

Reply:
Can't hurt to mention here.  I'm not thinking to change to the constants in File/IO, but it may be worth mentioning the difference there.